src/App.tsx

```diff
@@ -15,71 +16,92 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
```

Comment:
Replace with `goods[goods.indexOf('Jam')] || ''`, it will be better, cause if you don't have Jam in the array - it will apply an empty string as the default value

Reply:
Не зрозумів цього покращення

Reply:
просто для кейсу, якщо раптом в тебе не буде `Jam` в масив, то й не потрібно його відображати
тому аби цього уникнути, можна спочатку знайти його по індексу, а якщо немає такого, то встановити пусту строку